hacks/player properties/player_rendersize_recoded.xml

```diff
@@ -0,0 +1,256 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<hack name="Player Size" author="SlightlyInteligentMonkey">
```

Comment:
ive seen this spelled with 1 `l` or 2 in different files so I don't really know which is right but might wanna check em over